package/CHANGELOG

```diff
@@ -117,6 +117,7 @@ Fixes
   * new `Results` class now can be pickled/unpickled. (PR #3309) 
 
 Enhancements
+  * MOL2 parser populates elements attribute from SYMBL atom types (Issue #3062)
```

Comment:
Is it SYMBL or SYBYL atom types? (just looking at the manual, I thought it was the latter - pages 52-53?)

```suggestion
  * MOL2 parser populates elements attribute from SYBYL atom types (Issue #3062)
```